### DISCUSSION TITLE
Certificate for covid-scenarios.org and staging are invalid

### DISCUSSION THREAD

**Discussion by Nectarineimp** [neherlab/covid19_scenarios#664] [General]
I'd love to look at the site created from this codebase, but I don't know if it has been hacked or not. Is anyone aware of this? Thanks! Looks great, by the way.

**Comment by ivan-aksamentov** [comment]
@Nectarineimp Hi, Peter.

We had a similar report last month: https://github.com/neherlab/covid19_scenarios/issues/516 and I think we ended up on local internet setup or VPN issue, because the domain name was resolving to some strange IP address.

Our app is static (just a bunch of files) and is served statically from a relatively vanilla AWS setup (S3 + Cloudfront + Route53) with Amazon's certificates, so there isn't really anything to hack. 

The app is working fine for us (maintainers). I tried these tools and they were unable to detect any problems either:
https://www.sslshopper.com/ssl-checker.html#hostname=https://covid19-scenarios.org/
https://www.geocerts.com/ssl-checker
https://www.httpcs.com/en/test-ssl-certificate
https://ssltools.godaddy.com/views/certChecker

Could you please let me know:

 - [ ] what browser(s) have you tried?
 - [ ] what the "Certificate Details" window show?
 - [ ] what location are you connecting from? (Cloudfront distributes the served files in different server locations around the world)
 - [ ] can you see if your AWS locations had any disruptions recently: https://status.aws.amazon.com/ ?

If you are on Linux or Mac, could you please provide the results of running these commands:

 - [ ] To see what your DNS resolved the domain name to:

```
dig +short covid19-scenarios.org
```

 - [ ] 3 different commands to fetch the certificates and retrieve various information from them:
```
curl --insecure -v https://covid19-scenarios.org 2>&1 | awk 'BEGIN { cert=0 } /^\* SSL connection/ { cert=1 } /^\*/ { if (cert) print }'

openssl s_client -showcerts -connect covid19-scenarios.org:443

nmap -p 443 --script ssl-cert covid19-scenarios.org

```
 - [ ] path in the internet from your computer to whichever computer you are trying to connect (delete first couple of lines to not show your IP address):

```
traceroute covid19-scenarios.org
```



---

Meanwhile, you may also try our deployment to  now.sh / vercel:
https://covid19-scenarios.covid19-scenarios.now.sh/
But this is master branch, with latest features, but might be unstable. Not recommended for any serious work.
